static/app/utils/profiling/flamegraph/selectNearestFrame.spec.ts

```diff
@@ -0,0 +1,104 @@
+import {selectNearestFrame} from './selectNearestFrame';
+
+class Node {
```

Comment:
I don't think we need this anymore right - we should be doing this using the flamegraph class or the frame class